app/models/transfer/between_plates.rb

```diff
@@ -75,7 +75,16 @@ def calculate_location_submissions
 
     destination_sources.each_with_object({}) do |dest_source, store|
       dest_loc, sources = *dest_source
-      uuid, transfer_details = pre_cap_groups.detect { |_uuid, group_details| group_details[:wells].sort == sources.sort }
+
+      found_precap_groups = pre_cap_groups.select { |_uuid, group_details| group_details[:wells].sort == sources.sort }
```

Comment:
I would put an underscore in the middle of 'pre' and 'cap' to make it consistent with 'pre_cap_groups' - might save a developer some typos in the future :)